src/components/Comments/ReplyCommentItem.js

```diff
@@ -103,11 +103,11 @@ const ReplyCommentItem = ({
     }
   };
 
-  const onBlock = (commentBlock) => {
+  const onBlock = () => {
     refBlockComponent.current.openBlockComponent({
       anonimity: false,
-      actor: commentBlock.user,
-      id: commentBlock.id
+      actor: comment.user,
+      id: comment.id
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onBlock` function has been modified to no longer take a parameter. This change could potentially introduce bugs if there are other parts of the code that still call `onBlock` with a parameter expecting the old behavior. Please ensure that all calls to this function have been updated accordingly.

```diff
-  const onBlock = (commentBlock) => {
+  const onBlock = () => {
     refBlockComponent.current.openBlockComponent({
       anonimity: false,
-      actor: commentBlock.user,
-      id: commentBlock.id
+      actor: comment.user,
+      id: comment.id
     });
   };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->